.github/workflows/stale-prs.yml

```diff
@@ -0,0 +1,113 @@
+name: Mark stale pull requests in Shepherd repo
+
+on:
+  schedule:
+  - cron: '0 0 * * *'
+
+jobs:
+  stale:
+
+    runs-on: ubuntu-latest
+    steps:      
+      - name: Checkout code
+        uses: actions/checkout@ee0669bd1cc54295c223e0bb666b733df41de1c5
+        
+      - name: Check PRs and compile report
+        id: compile_report
+        uses: actions/github-script@211cb3fefb35a799baa5156f9321bb774fe56294
```

Comment:
Our internal guidelines say. So i think what I have in my code is the way to go? 

> Pin the Action's version to a full-length commit ID instead of a tag.

Reply:
https://github.com/rancher/security-team/blob/main/docs/rancher-gha-standards.md#security-recommendations